Formula/fstrm.rb

```diff
@@ -33,7 +32,8 @@ def install
   end
 
   test do
-    system "sh", "-c", "#{bin}/fstrm_capture -t protobuf:dnstap.Dnstap -u dnstap.sock -w capture.fstrm -dddd & sleep 1; kill $!"
+    system "sh", "-c", "#{bin}/fstrm_capture -t protobuf:dnstap.Dnstap -u \
```

Comment:
That's a good question actually. I presumed OP double checked this but easy enough to check again.